client/my-sites/activity/activity-log/style.scss

```diff
@@ -27,15 +26,15 @@
 		bottom: 0;
 		height: 16px;
 		z-index: -1;
-		background: blue;
-		background: linear-gradient( transparentize( $gray-light, 1 ), $gray-light );
+		background: var( --color-neutral-100 );
+		background: linear-gradient( var( --color-neutral-100 ), var( --color-surface-backdrop ) );
```

Comment:
instead of going from a transparent backdrop to the backdrop, go from the source color to the backdrop